src/test/testType.ts

```diff
@@ -63,10 +63,11 @@ export class TestTypeImpl {
       throw errorWithCallLocation(`test() can only be called in a test file`);
     const location = callLocation(suite.file);
 
-    const ordinalInFile = countByFile.get(suite.file) || 0;
-    countByFile.set(location.file, ordinalInFile + 1);
```

Comment:
this mismatch with `suite.file` and `location.file` was just broken before right?

Reply:
Yeah...